.github/workflows/pull_request.yml

```diff
@@ -15,8 +15,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          fetch-depth: 0
-          token: ${{ secrets.GH_PAT }}
+          fetch-depth: 100
```

Comment:
I think we were getting an error here because this repo is older and quite large. Running into rate limits of some kind, I think.

Hopefully 100 commits of history is enough to merge any PRs - let me know if you disagree!